[----------] Running tests from MockTest
[ RUN      ] MockTest.ExpectationSatisfied
[       OK ] MockTest.ExpectationSatisfied
[ RUN      ] MockTest.MockExpectationNotSatisfied
/some/path/mock_test.go:56:
Unsatisfied expectation; expected At to be called at least 1 times; called 0 times.

[  FAILED  ] MockTest.MockExpectationNotSatisfied
[ RUN      ] MockTest.ExpectCallForUnknownMethod
/some/path/mock_test.go:61:
Unknown method: FooBar

[  FAILED  ] MockTest.ExpectCallForUnknownMethod
[ RUN      ] MockTest.UnexpectedCall
/some/path/mock_test.go:65:
Unexpected call to At with args: [11 23]

[  FAILED  ] MockTest.UnexpectedCall
[ RUN      ] MockTest.InvokeFunction
[       OK ] MockTest.InvokeFunction
[----------] Finished with tests from MockTest
--- FAIL: TestSomething (1.23s)
FAIL
exit status 1
FAIL somepkg 1.234s
